api/v1/_submissions/BackendSubmissionsHandler.inc.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file api/v1/_submissions/BackendSubmissionsHandler.inc.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2003-2016 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class BackendSubmissionsHandler
+ * @ingroup api_v1_backend
+ *
+ * @brief Handle API requests for backend operations.
+ *
+ */
+
+import('lib.pkp.api.v1._submissions.PKPBackendSubmissionsHandler');
+
+class BackendSubmissionsHandler extends PKPBackendSubmissionsHandler {
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		\HookRegistry::register('API::_submissions::params', array($this, 'addAppSubmissionsParams'));
```

Comment:
Hmm, wouldn't a regular super/subclass function call handle this better? Hooks are spaghetti.